src/DotLiquid.Tests/FileSystemTests.cs

```diff
@@ -22,22 +53,20 @@ public void TestDefault()
         public void TestLocal()
         {
             LocalFileSystem fileSystem = new LocalFileSystem(@"D:\Some\Path");
-            Assert.AreEqual(@"D:\Some\Path\_mypartial.liquid", fileSystem.FullPath("mypartial"));
-            Assert.AreEqual(@"D:\Some\Path\dir\_mypartial.liquid", fileSystem.FullPath("dir/mypartial"));
+            foreach (var validPath in validPaths)
+                Assert.AreEqual(System.IO.Path.Combine(@"D:\Some\Path", validPath.Value.Replace('\\', Path.DirectorySeparatorChar)), fileSystem.FullPath(validPath.Key));

```

Comment:
System.IO is redundant, or do we need to disambiguate because we defined another Path class conflicting with the system?

Reply:
Also preferable to use named parameters with Assert.Equal to avoid non-obvious out of order parameters.